infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -1219,7 +1219,18 @@ public void close() {
             fh.close();
         }
 
+        for (FileChannel channel : channelsToSync) {
+            try {
+                channel.force(true);
+            } catch (IOException e) {
+                log.debug("Can't force updates in the channel {}", e.getMessage());
+            } finally {
+                IOUtils.closeQuietly(channel);
+            }
+        }
+
```

Comment:
`channelsToSync` is a subset of of `writeChannels` what's the point of closing them again ?

Reply:
Because it's a pretty fragile structure - having two collections that could or couldn't contain same elements(instances of filechannels). 

Since it's not guaranteed at compile time that `writeChannels` is always a superset of `channelsToSync` it's safe to close those descriptors (just in case) even if we know that writeChannels should be a superset of channelsToSync.

However, I'm ok not to add this change.

Reply:
removed